ros_ws/src/common/constants.h

```diff
@@ -59,6 +67,10 @@ const std::string TELEOP_NODE = "teleop_control";
  * @brief The node name for the teleop_mux node
  */
 const std::string TELEOP_MUX_NODE = "teleop_mux_node";
+/**
+ * @brief The node name for the motor_controller node
+ */
+const std::string MOTOR_CONTROLLER_NODE = "motor_controller_node";
```

Comment:
Now that I think about this a little bit it might actually make sense to have the node names be literals.  The way we have it right now makes the entire workspace necessary to run a package.  I think might be a good idea to follow the ROS idea that packages should be atomic and interchangeable regardless of workspace.